websites/A/atomicradio/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"name": "VocalZero",
+		"id": "223891083724193792"
+	},
+	"service": "atomicradio",
+	"description": {
+		"en": "The future of radio is here. Experience 12 Spaces with non-stop music you'll love."
+	},
+	"url": "atomic.radio",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/16lVdWy.png",
```

Comment:
Logo's must be 512x512 as per our [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).